docs/user-guide/migrating-from-jscs.md

```diff
@@ -0,0 +1,79 @@
+# Migrating from JSCS
+
+In April 2016, we [announced](http://eslint.org/blog/2016/04/welcoming-jscs-to-eslint) that the JSCS project was shutting down and the JSCS team would be joining the ESLint team. This guide is intended to help those who are using JSCS to migrate their settings and projects to use ESLint. We've tried to automate as much of the conversion as possible, but there are some manual changes that are needed.
+
+## Terminology
```

Comment:
Although it was removed in [jscs 3.0](http://jscs.info/changelog#version-3-0-0-https-github-comjscs-devnode-jscscomparev2-11-0-v3-0-0-2016-04-14), we could mention that there is no `esnext` option and that there are `ecmaFeatures`

Some options are the same (--fix is the same).

We can mention the error levels (0,1,2 and "off", "warn", "error") vs in jscs where you just use `true` or `false` or `null`